WordPress/Classes/Categories/NSURLRequest+Helpers.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+
+@interface NSURLRequest (Helpers)
+
++ (NSURLRequest *)requestWithURL:(NSURL *)url userAgent:(NSString *)userAgent;
+
++ (NSURLRequest *)requestForAuthenticationWithURL:(NSURL *)loginUrl
```

Comment:
I'd consider documenting this class and these methods since they may be non-obvious to someone peeking through the code.